noir-projects/noir-contracts/contracts/new_key_registry_contract/src/main.nr

```diff
@@ -0,0 +1,60 @@
+contract NewKeyRegistry {
+    use dep::authwit::auth::assert_current_call_valid_authwit_public;
+
+    use dep::aztec::{
+        keys::{PublicKeys, stored_keys::StoredKeys}, state_vars::{PublicMutable, Map},
+        protocol_types::{point::Point, address::{AztecAddress, PartialAddress}}
+    };
+
+    #[aztec(storage)]
+    struct Storage {
+		current_keys: Map<AztecAddress, PublicMutable<StoredKeys>>,
+    }
+
+    impl<Context> Storage<Context> {
```

Comment:
We implement this manually to have a control over slot number used, right? If yes please add here a comment as it's not immediately clear.

Reply:
Ah good call. Yes, that's the reason.